unlock-app/src/__tests__/helpers/Locks.test.js

```diff
@@ -144,4 +144,25 @@ describe('lockHelpers', () => {
 
     expect(getLockStatusString(transactions, Lock.address)).toEqual('confirming')
   })
+
+  it('should return the name of a lock from its transaction', () => {
+    const Lock = {
+      address: '0x1234567890',
+    }
+    const transactions = {
+      all: {
+        theTransactionWereLookingFor: {
+          hash: 'theTransactionWereLookingFor',
+          confirmations: config.requiredConfirmations-1,
```

Comment:
Could you just use `config.requiredConfirmations` here so that we do not test for 2 things at once?